src/ray/gcs/actor_state_accessor.h

```diff
@@ -0,0 +1,35 @@
+#ifndef RAY_GCS_ACTOR_STATE_ACCESSOR_H
+#define RAY_GCS_ACTOR_STATE_ACCESSOR_H
+
+#include "ray/common/id.h"
+#include "ray/gcs/call_back.h"
+#include "ray/gcs/format/gcs_generated.h"
+
+namespace ray {
+
+namespace gcs {
+
+class AsyncGcsClient;
+
```

Comment:
Add some comments for this class? 

same for other classes

Reply:
sure